rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_7.json

```diff
@@ -0,0 +1,2768 @@
+{
+    "rule-19-7-a": {
+        "Section": 19,
+        "Rule": 7,
+        "Test": "a",
+        "test_description": "System 7 serving two zones that do not have laboratory spaces where baseline and proposed OA are identical.",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "19-7",
+            "ruleset_reference": "G3.1.2.5 and Exception 2",
+            "rule_description": "Minimum ventilation system outdoor air intake flow shall be the same for the proposed design and baseline building design except when any of the 4 exceptions defined in Section G3.1.2.5 are met.\nExceptions\n2. When designing systems in accordance with Standard 62.1, Section 6.2, \u201cVentilation Rate Procedure,\u201d reduced ventilation airflow rates may be calculated for each HVAC zone in the proposed design with a zone air distribution effectiveness (Ez) > 1.0 as defined by Standard 62.1, Table 6-2. Baseline ventilation airflow rates in those zones shall be calcu-lated using the proposed design Ventilation Rate Procedure calculation with the following change only. Zone air distribution effectiveness shall be changed to (Ez) = 1.0 in each zone having a zone air distribution effectiveness (Ez) > 1.0. Proposed design and baseline build-ing design Ventilation Rate Procedure calculations, as described in Standard 62.1, shall be submitted to the rating authority to claim credit for this exception. ",
+            "applicable_rmr": "Baseline RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "primary_rule": "Yes",
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "proposed": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Required Building Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "VAV Air Terminal 1",
```

Comment:
Could you add the ```minimum_outdoor_airflow_multiplier_schedule``` key? This is because it's needed in the ```get_min_oa_cfm_sch_zone``` function.

Reply:
`minimum_outdoor_airflow_multiplier_schedule` added to all cases